libs/sdk-ui-kit/src/utils/featureFlags.ts

```diff
@@ -0,0 +1,44 @@
+// (C) 2021 GoodData Corporation
+import { IFeatureFlags } from "@gooddata/goodstrap/lib/@next/Header/types";
+
+const FREE = "free";
+const GROWTH = "growth";
+
+/**
+ * @internal
+ */
+export function shouldHidePPExperience(featureFlags: IFeatureFlags): boolean {
+    const hidePPExperience = featureFlags.hidePixelPerfectExperience as boolean;
```

Comment:
with every 'as' typing small bear dies ;), use real type conversion if needed, or type the local variable

Reply:
I use real type conversion instead of "as" typings thanks